### PR TITLE
T29 create tag

### PR DESCRIPTION
**Pull request by jimytz**

### CODE REVIEW

**Comment by adrianovi** [comment]
One of your scenarios is failing, the pipeline is falling because of that

File: src/test/resources/features/posts/tags.feature [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    Examples:
+      | User Role     | Status Line     |
+      | administrator | HTTP/1.1 200 OK |
+
+  @CreateATag @Smoke
+  Scenario Outline: A user with proper role should be able to create a tag
+    Given I am authorized with a user with "<User Role>" role
+    When I make a request to create a tag with the following query params
+      | name                    |
```
Comment (adrianovi) [inline src/test/resources/features/posts/tags.feature:20]: fix the spaces for "|" here

Comment (jimytz) [reply src/test/resources/features/posts/tags.feature]: IT is fixed now! thanks!

**Review by adrianovi — APPROVED** [review]
Nice!

**Pull request merged**